manifests/y/yuma14/ShutdownCountdown/1.1.0/yuma14.ShutdownCountdown.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: yuma14.ShutdownCountdown
+PackageVersion: 1.1.0
+Installers:
+- InstallerLocale: ja-JP
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/yuma140902/ShutdownCountdown/releases/download/v1.1.0/shutdowncountdown-installer-x64.msi
+  InstallerSha256: A9E720D0BA59ACA862EF987948C8DA5A221D1FF4CC37CA3C637BF57D194A6DF5
+  ProductCode: '{182B9474-1E96-4CAD-96B8-DAC2579D09AC}'

```

```suggestion
  ProductCode: '{45D0EDE9-1F66-4D01-A897-C96694725972}'
```
ProductCode for new installer.